.yarn/patches/@metamask-assets-controllers-patch-a3b39b55a6.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/dist/chunk-6W5EQ3JQ.js b/dist/chunk-6W5EQ3JQ.js
```

Comment:
This patch is no longer needed and could be removed.  It was hotfixed to v12.0.5, but 12.1.0 is on version 33 of assets-controllers which already contains this.

Reply:
https://github.com/MetaMask/metamask-extension/pull/26519/commits/48fbcd59a7283015a486905814ac526939230cf5 wasn't quite right.  It should have just deleted the `...a3b39b55a6.patch` file, but it additionally removed a necessary 12.1.0 patch `...3e7448c4cd.patch` from package.json.  This commit should be applied:  https://github.com/MetaMask/metamask-extension/commit/99e52f141a931ec3a10f691bf828f1d4bc8a492d

Reply:
Suggested commit applied with https://github.com/MetaMask/metamask-extension/pull/26519/commits/ee890f9e16808bec0821a4e6073de1817b64799f